hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestOzoneDebugShell.java

```diff
@@ -102,18 +104,20 @@ public void testChunkInfoCmdBeforeAfterCloseContainer(boolean isEcKey) throws Ex
     assertEquals(0, exitCode);
   }
 
-  @Test
-  public void testChunkInfoVerifyPathsAreDifferent() throws Exception {
+  @ParameterizedTest
+  @ValueSource(strings = {"FILE_SYSTEM_OPTIMIZED", "LEGACY", "OBJECT_STORE"})
+  public void testChunkInfoVerifyPathsAreDifferent(BucketLayout layout) throws Exception {
     final String volumeName = UUID.randomUUID().toString();
     final String bucketName = UUID.randomUUID().toString();
     final String keyName = UUID.randomUUID().toString();
-    writeKey(volumeName, bucketName, keyName, false);
+    writeKey(volumeName, bucketName, keyName, false, layout);
     int exitCode = runChunkInfoAndVerifyPaths(volumeName, bucketName, keyName);
     assertEquals(0, exitCode);
   }
 
-  @Test
-  public void testLdbCliForOzoneSnapshot() throws Exception {
+  @ParameterizedTest
+  @CsvSource(value = {"FILE_SYSTEM_OPTIMIZED,fileTable", "LEGACY,keyTable", "OBJECT_STORE,keyTable"})
```

Comment:
Please use `@EnumSource`, and call `OMMetadataManager.getKeyTable(layout).getName()` to get the table name.

https://github.com/apache/ozone/blob/abc1e028345161444809f0b8815de55399d0c157/hadoop-ozone/interface-storage/src/main/java/org/apache/hadoop/ozone/om/OMMetadataManager.java#L386

https://github.com/apache/ozone/blob/abc1e028345161444809f0b8815de55399d0c157/hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/Table.java#L176-L181